FullStoryMiddleware/FullStoryMiddleware.m

```diff
@@ -117,11 +117,15 @@ - (SEGPayload *)getNewPayloadWithFSURL:(SEGContext * _Nonnull)context {
             NSMutableDictionary *newProps = [[NSMutableDictionary alloc] initWithDictionary:screenPayload.properties];
             [newProps setValue:[FS currentSessionURL] forKey:@"FSSessionURL"];
 
-            newPayload = [[SEGScreenPayload alloc]
-                                           initWithName:screenPayload.name
-                                           properties:newProps
-                                           context:screenPayload.context
-                                           integrations:screenPayload.integrations];
+            // SEGScreenPayload initWithName signiture has changed @v4.1
+            // temporary workaround to set the catagory for post 4.1
+            SEGScreenPayload *screen = [[SEGScreenPayload alloc] initWithContext:screenPayload.context integrations:screenPayload.integrations];
+            [screen setValue:[screenPayload.name copy] forKey:@"name"];
+            [screen setValue:[newProps copy] forKey:@"properties"];
+            if(screen.category) {
```

Comment:
Did you mean to check `screenPayload.category`? Asking since that's the value you use below.

Also I'd give `if` some breathing room and add a space after. It's a bit confusing to read it that way since `if` isn't a function.